scripts/install_requirements.py

```diff
@@ -11,12 +11,20 @@
 
 from _utils import _LogWrapper
 
+python_version = sys.version.split(" ")[0][:4]
+
 _REQUIREMENTS_STEMS = ["requirements", "requirements-dev"]
 
 _INSERTION_FIXED = "-fixed"
 
 _REQUIREMENTS_EXTENSION = "txt"
 
+# Configuration for the --pinned argument including explicit package versions
+# that are minimally required for the latest Python versions.
+_PINNED_REQUIREMENTS = {
```

Comment:
Small point, but if this is just for `numpy`, should the name reflect that?

Reply:
yes true, it is from the initial code where it was all more general and more customisable. we could change it, but we can also leave so we could add something(and of course change the other line where numpy is explicit with it then). I think soon `torch` will become an issue for `3.13`, I am following their thread. what do you think?